src/App.js

```diff
@@ -83,11 +83,7 @@ function MainComponent() {
     // UX change removed 5 mile radius as an option so this will set cookies
     // previously set to 5 to the next smallest, 10
     // TODO: undo this snippet after sufficient time has passed
-    if (
-        filterCookies &&
-        filterCookies.hasOwnProperty("filterByZipCode") &&
-        filterCookies.filterByZipCode.miles === 5
-    ) {
+    if (filterCookies?.filterByZipCode.miles === 5) {
```

Comment:
Not the same code...  You need filterCookies?.filterByZipCode?.miles  (Note the second ?.)
The first ?. takes care of `if (filterCookies)`
The second ?. takes care of `filterCookies.hasOwnProperty("filterByZipCode")`


See [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)